### PR TITLE
154 add dropdown in button

### PR DESCRIPTION
**Pull request by imrochamatheus**

### CODE REVIEW

**Comment by iurynogueira** [comment]
Add output in button to get option selected in dropdown

**Comment by iurynogueira** [comment]
fix pipeline

**Comment by mizek1** [comment]
Check pipeline errors.
Apparently, in every component that button is declared/imported you'll need to declare badge and dropdown too.

File: projects/ion/src/lib/button/button.component.ts [hunk 1]
```diff
@@ -32,14 +42,42 @@ export class ButtonComponent {
...
   @Input() loading? = false;
   @Input() loadingMessage = 'Carregando...';
+  @Input() multiple? = false;
   @Input() iconType? = '';
   @Input() rightSideIcon? = false;
   @Input() circularButton? = false;
+  @Input() options?: DropdownItem[];
+  @Input() showDropdown? = false;
   @Output() ionOnClick? = new EventEmitter();
+  @Output() selected = new EventEmitter();
```
Comment (mizek1) [inline projects/ion/src/lib/button/button.component.ts:52]:
```suggestion
  @Output() selected = new EventEmitter<DropdownItem[]>();
```

**Review by mizek1 — CHANGES_REQUESTED** [review]
Good job!
Update your branch with main because there are new changes to dropdown and it may affect this features.

File: projects/ion/src/lib/button/button.component.spec.ts [hunk 2]
```diff
@@ -146,3 +148,100 @@ describe('load ButtonComponent', () => {
     expect(button.children[1].textContent).toContain(loadingMessage);
   });
 });
+
+describe('ButtonComponent with badge', () => {
```
[the remark this answers is not in the document]
Comment (imrochamatheus) [reply projects/ion/src/lib/button/button.component.spec.ts]: show, added!

**Review by iurynogueira — APPROVED** [review]

**Review by Deeved — APPROVED** [review]

**Pull request merged**